src/content/en/showcase/2017/santa.md

```diff
@@ -0,0 +1,217 @@
+project_path: /web/_project.yaml
+book_path: /web/showcase/_book.yaml
+
+{# wf_published_on: 2017-03-26T18:00:00.000Z #}
+{# wf_updated_on: 2017-03-26T18:00:00.000Z #}
+{# wf_featured_image: /web/showcase/2017/images/santa/featured.png #}
+{# wf_featured_snippet: Santa Tracker is a fun and educational experience for everyone—a holiday tradition here at Google. In 2016, we upgraded the Santa Tracker site to support offline via Service Worker. #}
+{# wf_tags: pwa,polymer,casestudy #}
+
+
+# Santa Tracker as a PWA {: .page-title }
+
+*By Sam Thorogood, Chief Santaware Engineer, Google—with contributions from Philipp Legner, Software Engineer, Google*
+
+<a class="button button-primary" href="https://santatracker.google.com/">
+  View the site
+</a>
+
+## Results
```

Comment:
I don't understand. I've just followed the format of the other showcase articles, which don't use that format, and all start with a `Results` section. Can you elaborate @jpmedley ?